application/gqlgen/post.go

```diff
@@ -589,3 +589,76 @@ func (r *mutationResolver) RemovePotentialProvider(ctx context.Context, postID,
 
 	return &post, nil
 }
+
+func (r *mutationResolver) MarkRequestAsDelivered(ctx context.Context, postID string) (*models.Post, error) {
```

Comment:
Can this call the mutation resolver for UpdatePostStatus? Not only would it reduce duplicate code, but keep tests cleaner and shorter. Anything we can do to trim test execution time would be good.

Reply:
Yep. I've refactored to avoid duplicate code. It does seem good, though, to keep the tests in there, in case some day we want to have those resolvers do different things.